app/assets/stylesheets/views/_topics.scss

```diff
@@ -1,5 +1,12 @@
 .topics-page {
 
+  // Topic beta labels have a lot of text. Limiting the width of the label
+  // prevents the lines to wrap under the label.
+  .topic-beta-label .govuk-beta-label span {
+    width: 75%;
+    vertical-align: top;
+  }
```

Comment:
If that's the case I might pick this up and see if I can get the banner to work better with multiline text @edds @tijmenb